Source/WebCore/html/HTMLAnchorElement.h

```diff
@@ -93,12 +93,12 @@ class HTMLAnchorElement : public HTMLElement, public URLDecomposition {
     void setActive(bool active, bool pause, Style::InvalidationScope) final;
     bool isURLAttribute(const Attribute&) const final;
     bool canStartSelection() const final;
-    String target() const override;
+    AtomString target() const override;
     int defaultTabIndex() const final;
     bool draggable() const final;
     bool isInteractiveContent() const final;
 
-    String effectiveTarget() const;
+    AtomString effectiveTarget() const;
```

Comment:
Yes, I think I can return a `const AtomString&`, will fix.

Reply:
Oh, turns out that I cannot. it relies on target() which is a virtual function and returns an AtomString. I tried to make target() return an `const AtomString` but I could't because of SVGAElement::target():
`AtomString target() const final { return AtomString { m_target->currentValue() }; }`